### ISSUE TITLE
Segmentation of objects in bin

### ISSUE THREAD

**Issue by wkentaro** [start-jsk/jsk_apc#1364]

**Comment by yuyu2172** [comment]
![apc](https://cloud.githubusercontent.com/assets/2062128/14916214/1903390c-0e53-11e6-955e-556a751395d6.png)


#1357 #1358 #1360  #1368 #1369 #1370 #1372

**Comment by yuyu2172** [comment]
I debugged my code using the following launch code.

```
<launch>
  <arg name="launch_main" default="true" />
  <arg name="json" default="$(find jsk_2015_05_baxter_apc)/json/layout_12.json"/>


  <!-- set ros param -->  
  <node pkg="jsk_apc2016_common" type="publish_bin_info.py"
    name="set_bin_param" output="screen">
    <param name="json" value="$(arg json)"/>
    <rosparam file="$(find jsk_apc2016_common)/config/bin_shelf.yaml" command="load"/>
  </node>

  <arg name="INPUT_POINTS" value="/kinect2_torso/hd/points"/>
  <arg name="INPUT_INFO" value="/kinect2_torso/hd/camera_info"/>
  <arg name="INPUT_IMAGE" value="/kinect2_torso/hd/image_color"/>
  <arg name="INPUT_BIN_ARRAY" value="/set_bin_param/bin_array"/>
  <arg name="INPUT_TARGET" value="/right_hand/target_bin"/>

  <!-- note: initial_rot_list has been changed from 2015 to be compatiable 
              with Quaterniona-->

  <param name="/target_bin_name" value="g"/>

  <node name="segmentation_in_bin"
        pkg="jsk_2016_01_baxter_apc" 
        type="rbo_segmentation_in_bin_node.py" 
        output="screen">
    <remap from="~input" to="$(arg INPUT_POINTS)"/>
    <remap from="~input/info" to="$(arg INPUT_INFO)"/>
    <remap from="~input/image" to="$(arg INPUT_IMAGE)"/>
    <remap from="~input/bin_info_array" to="$(arg INPUT_BIN_ARRAY)"/>
    <remap from="~target_bin_name" to="$(arg INPUT_TARGET)"/>
    <param name="trained_pkl_path" value="$(find jsk_apc2016_common)/data/trained_segmenter.pkl" />
  </node>
</launch>
```

**Comment by yuyu2172** [comment]
Updated status on #1409

**Comment by wkentaro** [comment]
Closed via #1409
